starter-code/index.js

```diff
@@ -1,10 +1,49 @@
-var SortedList = function() {};
+var SortedList = function() {
+  this.items = [];
+  this.length = this.items.length;
+};
 
-SortedList.prototype.add     = function(item) {}
-SortedList.prototype.get     = function(pos) {}
-SortedList.prototype.max     = function() {}
-SortedList.prototype.min     = function() {}
-SortedList.prototype.average = function() {}
-SortedList.prototype.sum     = function() {}
+SortedList.prototype.add  = function (x) {
+  
+  this.items.push(x);
+  this.items = this.items.sort();
```

Comment:
Unfortunately, this will not sort an array of numbers correctly in all cases, because the default sort is according to unicode and not by size, so 100 comes before 90 for example. You should provide the sort method with your own sort function:
`this.items.sort(function(number1, number2) { return number1 - number 2})`